bottlecap/src/secrets/decrypt.rs

```diff
@@ -74,32 +139,28 @@ fn manual_decrypt(secret_arn: String, aws_config: AwsConfig) -> Result<String> {
         .text()
         .expect("Cannot deserialize body");
     let v: Value = serde_json::from_str(&body).expect("Failed to parse JSON");
-
-    return if let Some(secret_string) = v["SecretString"].as_str() {
-        Ok(secret_string.to_string())
-    } else {
-        Err(Error::new(std::io::ErrorKind::InvalidData, v.to_string()))
-    };
+    v
```

Comment:
Same comment as before, should we bubble all errors so we don't have expects here?

Reply:
done!